source/standards/alerting.html.md.erb

```diff
@@ -61,14 +63,15 @@ track alerts, escalate them (automatically or manually), acknowledge them
 
 ## Tools for alerting
 
-The default tool at GDS to be alerted of problems with your service
-is [PagerDuty](https://www.pagerduty.com/).
+Your monitoring system should use [PagerDuty](https://www.pagerduty.com/)
+to send alerts about problems with your service.
 
-We recommend using [Pingdom](https://www.pingdom.com/) as well
-in case PagerDuty becomes unavailable.
+We recommend using [Pingdom](https://www.pingdom.com/) as well, as a 
+top-level check on your service's availability.
```

Comment:
what do we mean by a top-level check?

Reply:
Can't think of a better way to phrase this - the point is that all Pingdom does is request a page on your website to check it's actually up, it doesn't do any detailed monitoring.